service/src/integrationTest/java/uk/gov/hmcts/reform/fpl/controllers/GeneratedOrderControllerTest.java

```diff
@@ -125,20 +144,63 @@ void shouldReturnErrorsWhenFamilymanNumberIsNotProvided() {
         assertThat(callbackResponse.getErrors()).containsExactly("Enter Familyman case number");
     }
 
-    @Test
-    void shouldTriggerOrderEventWhenSubmitted() throws Exception {
-        String expectedCaseReference = "19898989";
-        postSubmittedEvent(buildCallbackRequest());
+    @Nested
+    class Submitted {
```

Comment:
I'd want to split the whole class out at once rather than just take one type of callback out now and leave the rest - I can either add techdebt note or go ahead and split whole class (and I should do same for ActionCaseManagementOrderControllerTest in that case)

Reply:
no please don't follow wrong practice. Separate new code out, write comment for rest of code.

Reply:
okay, will split out 👍